get_tau_pet.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import pandas as pd
+
+'''
+We have a phenotypic spreadsheet with multiple visit dates per participant. We also have a spreadsheet with multiple
+tau pet values for (some) participants. The goal is to find each participant's baseline visit data from the phenotypic
+file, and then find a tau pet value that was collected within one year.
+'''
+
+# load baseline phenotypic data
+pheno = pd.read_csv('/Users/natashaclarke/Documents/SIMEXP/cwas_neuro_sandbox/adni_cwas/adni_spreadsheet.csv', index_col=0, header=0)
+
+# keep only the variables of interest
+pheno = pheno.filter(['Subject ID','Phase','Sex','Research Group', 'Visit','Study Date','Age'], axis=1)
+
+# convert 'study date' to 'session' in datetime format, to match other spreadsheets
+pheno['session'] = pd.to_datetime(pheno['Study Date'])
+
+# pull out only the subject id and asign it to the index
+pheno_subj = []
+for i in pheno['Subject ID']:
+    subj = i.split('_')[2].lstrip("0") # remove leading zeros since it won't match ADNI IDs
+    pheno_subj.append(subj)
+    
+pheno.index = pheno_subj
+pheno.rename_axis('RID',inplace=True)
+pheno.index = pheno.index.astype('int64')
+
+# separate patients and controls, because (in theory) we can use any control visit as baseline, but
+# for patients we want their actual baseline data
+ps = ['AD', 'EMCI', 'LMCI', 'MCI', 'SMC']
+pdf = pheno[pheno['Research Group'].isin(ps)] # df of patient diagnoses
+
+cdf = pheno.loc[pheno['Research Group'] == 'CN'] # df of control diagnoses
```

Comment:
```suggestion
control_df = pheno.loc[pheno['Research Group'] == 'CN'] # df of control diagnoses
```
Of course remember to change things in other places.